Orange/widgets/data/owfeatureconstructor.py

```diff
@@ -541,8 +524,14 @@ def apply(self):
 
         desc = list(self.featuremodel)
 
+        def validate(source):
+            try:
+                return validate_exp(ast.parse(source, mode="eval"))
```

Comment:
Could't we just eval the expression and see if it fails with SyntaxError while the rest is on the user?